Tools/ardupilotwaf/boards.py

```diff
@@ -226,7 +226,7 @@ def configure_env(self, cfg, env):
                 '-Werror=unused-but-set-variable'
             ]
             (major, minor, patchlevel) = cfg.env.CC_VERSION
-            if int(major) >= 5 and int(minor) > 1 and not self.with_uavcan:
+            if int(major) > 5 or (int(major) == 5 and int(minor) > 1):
```

Comment:
Is the removal of the UAVCAN check on purpose?

Reply:
Yes.  overrides were added IIRC.